DependencyInjection/DunglasApiExtension.php

```diff
@@ -30,7 +30,7 @@ class DunglasApiExtension extends Extension implements PrependExtensionInterface
      */
     public function prepend(ContainerBuilder $container)
     {
-        if (null !== ($frameworkConfiguration = $container->getExtensionConfig('framework'))) {
+        if (!empty($frameworkConfiguration = $container->getExtensionConfig('framework'))) {
```

Comment:
True but we put [PHP 5.5](https://github.com/dunglas/DunglasApiBundle/blob/master/composer.json#L16) as minimal requirement though